witchcraft-server/src/logging/mod.rs

```diff
@@ -109,8 +117,41 @@ pub async fn audit_log(entry: AuditLogEntry) -> Result<(), Error> {
     }
 }
 
-/// Blocking variant of [audit_log] that only returns once the the audit log has been
+/// Blocking variant of [audit_log] that only returns once the audit log has been
 /// successfully written or if the audit log has failed.
 pub fn audit_log_blocking(entry: AuditLogEntry) -> Result<(), Error> {
     block_on(audit_log(entry))
 }
+
+/// Writes the provided V2 event log entry using the standard logging appender.
+/// Returns an error if the global event logger is not initialized.
+///
+/// The returned future completes once the event log has been successfully written.
+pub async fn event_log(entry: EventLogV2) -> Result<(), Error> {
```

Comment:
I think for event logs we can have this be more of a fire-and-forget setup than audit, where we need to ensure that the entry has been successfully written before continuing. It could look something like this: https://github.com/palantir/witchcraft-rust-server/blob/develop/witchcraft-server/src/logging/service.rs#L202-L205